### PR TITLE
Feature/us975 rsvp yes

### PR DESCRIPTION
**Pull request by silbermm**
Please reveiw

### CODE REVIEW

File: Gateway/crds-angular.test/Services/ServeServiceTest.cs [hunk 1]
```diff
@@ -315,6 +318,96 @@ public void GetMyFamiliesServingEventsTest()
             Assert.AreEqual("10:00:00", servingTime.Time);
         }
 
+<<<<<<< HEAD
```
Comment (IngageBrewerM) [inline Gateway/crds-angular.test/Services/ServeServiceTest.cs]: Missed merge tags

File: crossroads.net/e2e/pages/login_page.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+var env = require("../environment");
```
Comment (IngageBrewerM) [inline crossroads.net/e2e/pages/login_page.js]: Filename should contain e2e

File: crossroads.net/e2e/pages/myserve_page.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+var env = require("../environment");
```
Comment (IngageBrewerM) [inline crossroads.net/e2e/pages/myserve_page.js]: Filename should contain e2e

File: crossroads.net/e2e/pages/login_page.js (hunk 2 above)
Comment (silbermm) [inline crossroads.net/e2e/pages/login_page.js]: This is a 'support' file for the testing framework, not an e2e test

File: crossroads.net/e2e/pages/myserve_page.js (hunk 3 above)
Comment (silbermm) [inline crossroads.net/e2e/pages/myserve_page.js]: This is a 'support' file for the testing framework, not an e2e test

**Comment by IngageBrewerM** [comment]
:+1:

**Pull request merged**